ui/src/app/store/machine/types/enum.ts

```diff
@@ -20,6 +20,31 @@ export enum DiskTypes {
   VOLUME_GROUP = "lvm-vg",
 }
 
+export const MachineFormNames = {
+  ABORT: "abort",
+  ACQUIRE: "acquire",
+  ADD: "add",
+  ADD_CHASSIS: "add-chassis",
```

Comment:
We can't just use `MachineActions` due to these extra two, which aren't tied to a machine's actions.